src/server/enrollment-system.js

```diff
@@ -341,9 +341,6 @@ function childToDependentInfo(child) {
  */
 function childToDependentFinancialsInfo(child) {
   const incomes = resourceToIncomeCollection(child);
-  if (!incomes) {
```

Comment:
Hm, I do wonder if it's entirely dependent on the vet's financials specifically, or if it's conditional on _any_ financials being present. Would you be ok merging this in and following up/refining with him so they can get some progress on testing?